shared/tracker/header.render.desktop.js

```diff
@@ -16,51 +17,18 @@ export default class HeaderRender extends Component {
   }
 
   render () {
-    let headerStyle = this.props.currentlyFollowing ? styleHeaderFollowing : styleHeaderNotFollowing
-    let headerTextStyle = styleHeaderTextNormal
+    const isWarningAboutTrackerShowingUpLater = this.props.loggedin && !this.props.currentlyfollowing && this.state.showCloseWarning
+    const headerText = isWarningAboutTrackerShowingUpLater ? 'You will see this window every time you access this folder.' : this.props.reason
 
-    if (this.props.currentlyFollowing) {
-      switch (this.props.trackerState) {
-        case 'warning':
-          headerStyle = styleHeaderWarning
-          headerTextStyle = styleHeaderTextWarning
-          break
-        case 'error': headerStyle = styleHeaderError; break
-      }
-    }
-
-    let headerText: string = this.props.reason
-    let isWarning = false
-    if (this.props.loggedIn && !this.props.currentlyFollowing && this.state.showCloseWarning) {
-      isWarning = true
-      headerStyle = styleHeaderWarning
-      headerTextStyle = styleHeaderTextWarning
-      headerText = 'You will see this window every time you access this folder.'
-    }
-
-    // If there's a lastAction, it overrides everything else.
-    switch (this.props.lastAction) {
```

Comment:
Why was this `lastAction` display logic removed? Does this update the UI properly when the state of the tracker is changed? (after the user follows/unfollows/etc)